inference-engine/src/plugin_api/cpp_interfaces/impl/ie_infer_async_request_thread_safe_default.hpp

```diff
@@ -40,7 +40,7 @@ namespace InferenceEngine {
  *
  * @snippet example_async_infer_request.cpp async_infer_request:define_pipeline
  */
-class AsyncInferRequestThreadSafeDefault : public AsyncInferRequestThreadSafeInternal {
+class AsyncInferRequestThreadSafeDefault : public IAsyncInferRequestInternal {
```

Comment:
looks like we need to change HDDL plugin as well and test with product-config

Reply:
will be done